### PR TITLE
Simplify markdown output

### PR DESCRIPTION
**Pull request by danthareja**
Comment space is precious, and a verbose comment can be hard to parse (and sometimes lead to errors reactorcore-labs/app.sweat-shop.taser#17). This PR simplifies the markdown output to get students the most important information up front.
### Before

screen shot 2016-05-03 at 4 55 48 pm
### After

screen shot 2016-05-03 at 4 56 53 pm

### CODE REVIEW

File: lib/taser-markdown-report.js [hunk 1]
```diff
@@ -47,53 +45,36 @@ class TaserMarkdownReport {
...
 }
 
 function testReportHeader(report) {
-    let contextName = report.browser.name;
-    let numberOfErrors = report.errors.length;
     let numberOfPasses = report.testResults.passed.length;
     let numberOfFailures = report.testResults.failed.length;
 
     return [
-        `**Browser**: ${contextName}`,
```
Comment (danthareja) [inline lib/taser-markdown-report.js:90]: Browser has gone away because it's only available in half of the test cases. Any server-side test will display "none" for this, making it unnecessary (and potentially confusing) 50% of the time

**Comment by danthareja** [comment]
If this gets merged, please add a new release for `1.1.0` so it's dependents can be updated

**Comment by johndvorakHR** [comment]
ynspbdkv
giphy
ynspbdkv
